Amicabilibus+RemoveDuplicates/src/Amicabilibus.java

```diff
@@ -3,7 +3,7 @@
 
 public class Amicabilibus {
 	
-	public static final int upperbound = 901425; //(int) Math.pow(10, 6);
+	public static final int upperbound = (int) Math.pow(10, 6);
 	public static ArrayList<Integer> factorSums = new ArrayList<Integer>();
 	public static boolean[] checked = new boolean[upperbound+1];
```

Comment:
again, u don't need this boolean array here as it would significantly decrease the efficiency. Pls change